pegasus/sites.v3/hourofcode.com/public/partners.md

```diff
@@ -2,7 +2,11 @@
 title: <%= hoc_s(:title_partners) %>
 ---
 
-<%= view :signup_button %>
+The Hour of Code is driven by the Hour of Code and Computer Science Education Week Advisory and Review Committees. Representatives from K-12, academia, nonprofits, for-profits, and international organizations compose the [Advisory Committee](<%= resolve_url('/advisory-committee') %>) that has guided the strategy of the Hour of Code campaign.
```

Comment:
Seems right to me.

http://www.merriam-webster.com/dictionary/for-profit
http://www.merriam-webster.com/dictionary/nonprofit